app/assets/stylesheets/alchemy/sitemap.scss

```diff
@@ -210,22 +218,28 @@ $sitemap-url-xlarge-width: 350px;
   .page_urlname {
     display: none;
     margin-left: auto;
+    padding-left: $default-padding;
 
     @media screen and (min-width: $large-screen-break-point) {
       display: block;
-      width: $sitemap-url-large-width;
+      width: $sitemap-url-large-width + 110;
```

Comment:
Why are these numbers? Should the 110 also be a variable?

Reply:
Good point. Added a variable.